WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -2,6 +2,7 @@ import Foundation
 import CocoaLumberjack
 import WordPressShared
 import WordPressFlux
+import UIKit
```

Comment:
We could remove this new import from the changes if it is not needed 🙏

Reply:
It is needed for UIPasteboard. If we will remove it, UIPasteboard will not work.

Reply:
I missed that 😬 
Thank you for the clarification @mertGokduman 🙇